pkgs/applications/editors/jetbrains/update.pl

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env perl
+#!/usr/bin/env nix-shell
+#!nix-shell -i perl -p perl perlPackages.LWP
```

Comment:
oops, forgot about LWP (I have a lot if perl modules in configuration.nix)
Does it work with https urls without ```perlPackages.LWPProtocolhttps``` ?